tests/plugins/pastefromword/generated/meta_generator.js

```diff
@@ -0,0 +1,34 @@
+/* bender-tags: clipboard,pastefromword */
```

Comment:
Not a best idea to put a full additional test suite for a single case, instead we could put it into `tests/plugins/pastefromword/generated/excel.js`. It can be a separate TC, but let's avoid spamming too much suites.